src/structlog/_loggers.py

```diff
@@ -12,6 +12,7 @@
 import threading
 
 from pickle import PicklingError
+from sys import stderr, stdout
```

Comment:
Why this additionally to sys above?

Reply:
If you do `sys.stdout`, `print(file=None if self._file is sys.stdout else self._file` won't work properly when `sys.stdout` gets patched (by rich, for example).

Reply:
Because `self._file` will be unpatched, and `sys.stdout` will be `<rich.file_proxy.FileProxy object at 0x12407aac0>`. And also it's an additional attribute access, who wants that right?

Reply:
Could you please add a test that proves this property then?

Reply:
Yessir.